lms/templates/course_modes/track_selection.html

```diff
@@ -0,0 +1,193 @@
+            if(width <= 768) {
+                $('.collapsible-item').css({"display":"none"});
+                $('.collapsible').css({"display":"block"});
+            };
+        }
+        window.addEventListener("resize", onresize);
+
+        // responsive: show more 
+        $(function(){
+            if($(window).width() <= 768) {
+            $('.collapsible-item').css({"display":"none"});
+            $('.collapsible').css({"display":"block"});
+            };
+        });
+        $(function(){
+            $('.collapsible').click(function(){
+                $('.collapsible').css({"display":"none"});
+            $('.collapsible-item').css({"display":"list-item"});
+            }); 
+        });
+    </script>
+</%block>
+<%static:webpack entry="Currency">
+    new Currency();
+</%static:webpack>
+
+<%block name="header_extras">
+    <link rel="stylesheet" type="text/css" href="${static.url('paragon/static/paragon.min.css')}" />
+</%block>
+
+<%block name="content">
+    <div id="currency_data" value="${currency_data}"></div>
+
+    <div class="container-md container-fluid mx-auto">
+        <section class="wrapper m-4 m-md-0" dir="ltr"> 
+            <div class="row">
+                <header>
+                    <h3 class="py-3 pt-md-5 pb-md-4">Choose a path for your course in ${course_name}</h3>
+                </header>
+                <form class="form-track-selection" method="post" name="enrollment_mode_form" id="enrollment_mode_form">
+                    <%
+                    b_tag_kwargs = {'b_start': HTML('<b>'), 'b_end': HTML('</b>')}
+                    %>
+                    % if "verified" in modes:
+                    <div class="track-selection-choice track-selection-certificate col-6 mb-5">
+                        <div class="certificate-container">
+                            <div class="triangle-overlay"></div>
+                            <span class="trophy-icon">
+                                <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M26.6667 3.33333H23.3333V0H6.66667V3.33333H3.33333C1.5 3.33333 0 4.83333 0 6.66667V8.33333C0 12.5833 3.2 16.05 7.31667 16.5667C8.36667 19.0667 10.6167 20.95 13.3333 21.5V26.6667H6.66667V30H23.3333V26.6667H16.6667V21.5C19.3833 20.95 21.6333 19.0667 22.6833 16.5667C26.8 16.05 30 12.5833 30 8.33333V6.66667C30 4.83333 28.5 3.33333 26.6667 3.33333ZM3.33333 8.33333V6.66667H6.66667V13.0333C4.73333 12.3333 3.33333 10.5 3.33333 8.33333ZM26.6667 8.33333C26.6667 10.5 25.2667 12.3333 23.3333 13.0333V6.66667H26.6667V8.33333Z" fill="#EFF8FA"/>
+                                </svg>
+                            </span>
+                            <p class="price-display">${currency_symbol}${min_price} ${currency}</p>
+                            <div class="choice-title"><h4>${_("Earn a certificate")}</h4></div>
+                            <div class="choice-bullets">
```

Comment:
I'm trying to remember- is the ${_("text goes here")} convention the way we signal a placeholder that'll be translated in edx-platform? I see a bunch of text that doesn't have that around it, so I wanted to confirm whether all the page's strings will be able to be translated

Reply:
Correct, coming in the final commit